include/yaramod/utils/trie.h

```diff
@@ -368,7 +368,7 @@ class Trie
 				nodes.push_back(subnode);
 		}
 
-		return result;
+		return {result.rbegin(), result.rend()};
```

Comment:
What about using `std::reverse` and reversing the `result` in place? I am not sure about RVO picking on this and eliminating unnecessary allocations.

Reply:
That is certainly a better idea, thank you and please have a look once again.

Reply:
LGTM now. And thanks for noticing and fixing it.